src/schema/v2/partner/partnerInquirerCollectorProfile.ts

```diff
@@ -11,6 +12,11 @@ const InquirerCollectorProfileFields: GraphQLFieldConfigMap<
   ResolverContext
 > = {
   ...CollectorProfileFields,
```

Comment:
Do you know why this type exists? It looks to be an exact duplicate of the `CollectorProfile` type.

Reply:
`UpdateCollectorProfile` uses it too without the extra properties we started exposing to `Partners` only. Basically `me` queries and mutations as `UpdateCollectorProfile`.